src/Microsoft.DotNet.Build.Tasks/Targets/Build.Common.targets

```diff
@@ -8,6 +8,8 @@
     <AssemblyVersion Condition="'$(AssemblyVersion)'==''">999.999.999.999</AssemblyVersion>
     <CLSCompliant Condition="'$(CLSCompliant)'=='' and '$(IsTestProject)'=='true'">false</CLSCompliant>
     <CLSCompliant Condition="'$(CLSCompliant)'==''">true</CLSCompliant>
+
+    <ExcludeLocalizationImport Condition="'$(ExcludeLocalizationImport)'=='' And !Exists('$(MSBuildProjectDirectory)\MultilingualResources\$(MSBuildProjectName).de.xlf')">true</ExcludeLocalizationImport>
```

Comment:
This condition checks if the project has been localized.  If it has the .de.xlf file, then I think we can assume that it has the .xlf files for all 9 languages.  The alternative would be to check for the existence of all 9, perhaps using an ItemGroup, but I don't really see any value in doing that way instead.  I don't think there's a use case yet for a project having xlf files in some languages but not others.